base/sparse/spqr.jl

```diff
@@ -309,6 +324,18 @@ function Base.getindex(F::QRSparse, d::Symbol)
     end
 end
 
+function Base.show(io::IO, mime::MIME{Symbol("text/plain")}, F::QRSparse)
+    println(io, summary(F))
```

Comment:
It is using [hits fallback](https://github.com/JuliaLang/julia/blob/master/base/show.jl#L1606) to write a string with the type information